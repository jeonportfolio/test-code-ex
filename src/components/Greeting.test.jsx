import { render } from "@testing-library/react";
import { describe, expect } from "vitest";
import Greeting from "./Greeting";

describe('Greeting Component', () => {
    it('should render correctly', () => {
       const { getByText } = render(<Greeting name={"brown"}/>);
       expect(getByText("Hello, brown")).toBeInTheDocument();
    });
    it('should render', () => {
        const { asFragment } = render(<Greeting name={"brown"}/>);
        expect(asFragment()).toMatchSnapshot();
    }) 
});