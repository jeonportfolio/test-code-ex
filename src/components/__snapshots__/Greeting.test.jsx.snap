// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Greeting Component > should render 1`] = `
<DocumentFragment>
  <h1>
    Hello, brown
  </h1>
</DocumentFragment>
`;
